pkg/kritis/secrets/secrets_test.go

```diff
@@ -81,3 +88,38 @@ func getTestSecret(namespace string, name string) (*v1.Secret, error) {
 	}
 	return nil, fmt.Errorf("Secret %s not found", name)
 }
+
+func createKeys(name string) (string, string) {
+	// Create a new pair of key
+	var key *openpgp.Entity
+	key, err := openpgp.NewEntity(name, "test", fmt.Sprintf("%s@grafeas.com", name), nil)
+	if err != nil {
+		glog.Errorf("Unexpected error: %v", err)
+	}
+	// Get Pem encoded Public Key
+	pub := getKey(key, openpgp.PublicKeyType)
+	// Get Pem encoded Private Key
+	priv := getKey(key, openpgp.PrivateKeyType)
+	return pub, priv
+}
+
+func getKey(key *openpgp.Entity, keyType string) string {
+	gotWriter := bytes.NewBuffer(nil)
+	wr, err := armor.Encode(gotWriter, keyType, nil)
+	if err != nil {
+		glog.Errorf("Unexpected error: %v", err)
```

Comment:
Errors usually aren't expected. Use the string to add useful context as to what function was called that reutrned the error, such as: glog.Errorf("encode failed: %v")

Reply:
good points, done